.vscode/settings.json

```diff
@@ -15,5 +15,6 @@
       72,
       88
     ]
-  }
+  },
+  "python.pythonPath": "C:\\Users\\mjenni03\\AppData\\Local\\Continuum\\anaconda3\\envs\\pmp\\python.exe"
```

Comment:
I don't understand why you need this in your set up? ...

Reply:
Sorry that was a mistaken inclusion.

VS Code auto adds this line everytime I activate my `pmp` environment. Most of the time this doesn't happen because `pmp` is my default in my User Settings. But I have other environments that I activate for work repos, and switching back from them to `pmp` triggers the addition of this line in the Workspace Settings. Every now and then I (embarrassingly) miss it =/

Reply:
I wonder how to make it not do that any more ...

Reply:
I don't think there is a (simple) way, from my Googling. But I have now implemented and tested a git filter that should prevent it from now on. See https://github.com/pymedphys/pymedphys/pull/437/commits/e5db2a11c077d6a30be36710cf2edbb228b42b73 and take a squiz in `.git/config` :)